src/Sulu/Bundle/MediaBundle/DependencyInjection/Configuration.php

```diff
@@ -79,7 +79,7 @@ public function getConfigTreeBuilder()
                     ->arrayNode('local')
                         ->addDefaultsIfNotSet()
                         ->children()
-                            ->scalarNode('path')->defaultValue('%kernel.root_dir%/../uploads/media')->end()
+                            ->scalarNode('path')->defaultValue('%kernel.var_dir%/uploads/media')->end()
```

Comment:
@danrot @chirimoya what do you think about `sulu.var_dir`? since its no a symfony parameter?